inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -920,3 +921,81 @@ def normalize_collaborations(obj, eng):
         collaboration['value'] = collaboration_normalized_name
         collaboration['record'] = response[0].self.to_dict()
     return obj
+
+
+def _match_lit_author_affiliation(raw_aff):
+    query = Q(
+        Q(
+            "nested",
+            path="authors",
+            query=(
+                Q("match", authors__raw_affiliations__value=raw_aff)
+                & Q("exists", field="authors.affiliations.value")
+            ),
+            inner_hits={},
+        )
+        & Q("term", _collections="Literature")
+        & Q("term", curated=True)
```

Comment:
these should be part of the [filter context](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-filter-context.html#filter-context), they're not relevant for scoring.